### PR TITLE
e2e: user creation by administrator

### PR DESCRIPTION
**Pull request by taohidkhan22**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/team/create-user-administrator.spec.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage()
+  })
+
+  test.afterAll(async () => {
+    await page.close()
+  })
+
+  test.describe('1.1 User creation started by national system admin', async () => {
```
Comment (Nil20) [inline e2e/testcases/team/create-user-administrator.spec.ts]:
```suggestion
  test.describe('1.1 User creation started by local system admin', async () => {
```

File: e2e/testcases/team/create-user-administrator.spec.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      await page.locator('#question-1').click()
+      await page.getByText(question01, { exact: true }).click()
+      await page.fill('#answer-1', 'Into the wild')
+      await page.locator('#question-2').click()
+      await page.getByText(question02, { exact: true }).click()
+      await page.fill('#answer-2', 'Burger')
+      await page.getByRole('button', { name: 'Continue' }).click()
+
+      await page.getByRole('button', { name: 'Confirm' }).click()
+      await expect(page.getByText('Account setup complete')).toBeVisible()
```
Comment (Nil20) [inline e2e/testcases/team/create-user-administrator.spec.ts]: We can extract them into a separate function as `e2e/testcases/team/create-user-national-system-admin.spec.ts` use the steps as well . Then use the function in both tests. Could be useful going forward as well.

**Review by Nil20 — APPROVED** [review]
